cli/src/lib/zksync-era-state.ts

```diff
@@ -106,11 +100,11 @@ export class ZksyncEraState {
     if (!this.data.protocolVersion) {
       throw new MissingRequiredProp("protocolVersion");
     }
-    const bytes = Buffer.from(hexToBytes(this.data.protocolVersion));
+    const bytes = Buffer.from(numberToBytes(this.data.protocolVersion, { size: 32 }));
```

Comment:
protocol version inside the contract is a uint256. I was treating is as a bytes32 as a mistake. There are several changes related to that fix like this one.